daprdocs/content/en/reference/components-reference/supported-secret-stores/kubernetes-secret-store.md

```diff
@@ -30,8 +30,11 @@ spec:
   type: secretstores.kubernetes
   version: v1
   metadata:
```

Comment:
This needs to be 
  metadata: []

see this as an example https://v1-8.docs.dapr.io/reference/components-reference/supported-state-stores/setup-inmemory/#component-format